x/nft/types/denom.go

```diff
@@ -7,6 +7,7 @@ import (
 )
 
 // NewDenom return a new denom
+// nolint: interfacer
```

Comment:
false positive?

Reply:
I get this lint error when I run `golangci-lint` locally. Anyway, `interfacer` is now deprecated (https://github.com/mvdan/interfacer). Should we remove this from github ci config?